cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -92,14 +93,36 @@ resource "aws_s3_bucket" "civiform_files_s3" {
   bucket = "civiform-files-s3"
 }
 
-resource "aws_s3_access_point" "civiform_files" {
+resource "aws_s3_bucket_public_access_block" "civiform_files_access" {
```

Comment:
i also think these should go in the generic aws folder and not the template folder